api/net/Areas/Subscriber/Controllers/FolderController.cs

```diff
@@ -123,20 +123,21 @@ public IActionResult Add(FolderModel model)
     /// Update folder for the specified 'id'.
     /// </summary>
     /// <param name="model"></param>
+    /// <param name="updateContent"></param>
     /// <returns></returns>
     [HttpPut("{id}")]
     [Produces(MediaTypeNames.Application.Json)]
     [ProducesResponseType(typeof(FolderModel), (int)HttpStatusCode.OK)]
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Folder" })]
-    public IActionResult Update(FolderModel model)
+    public IActionResult Update(FolderModel model, bool updateContent = false)
```

Comment:
We need this flag to allow us to update a folder without touching the content in it.